pulsar_spectra/catalogue_papers/Spiewak_2022.yaml

```diff
@@ -1,7 +1,7 @@
 {
  "J0023+0923": {
   "Frequency MHz": [
-   1284
```

Comment:
From the text: "All observations included in this paper were performed with
the MeerKAT L-band receiver (856–1 712 MHz) with the Pulsar
Timing User Supplied Equipement (PTUSE) back-end that uses
coherent dedispersion."
From this, the centre frequency is 1284 MHz and the 1400 MHz used on the tables is just an approximation to common values. If you agree with me, please change these values back to 1284.

Reply:
Yes, I changed it back. Thank you for pointing that out.